src/jupyter_contrib_nbextensions/nbextensions/hinterland/hinterland.js

```diff
@@ -93,14 +95,15 @@ define(function (require, exports, module) {
 							line: cur.line,
 							ch: cur.ch - 1
 						}, cur);
-
-						if (pre_cursor !== '' && !ignore_re.test(pre_cursor)) {
-							if (pre_cursor === '(') {
+						if (	pre_cursor !== '' &&
+								config.include_regexp.test(pre_cursor) &&
+								!config.exclude_regexp.test(pre_cursor) ) {
+							if (config.tooltip_regexp.test(pre_cursor)) {
 								cell.tooltip.request(cell);
```

Comment:
The `tooltip_regexp` character(s) must be included in the `include_regexp` characters, otherwise this is not fired. Perhaps that the the two regexp have to be merged (or more simply, the test modified perhaps  `(config.include_regexp.test(pre_cursor)  || config.tooltip_regexp.test(pre_cursor) ) && ..`  (untested)